lessons/04-client-jquery-ajax/HOMEWORK.md

```diff
@@ -0,0 +1,11 @@
+### Class 4 Homework
+For the homework, you will be integrating mongoDB into your Cat App from last class using mongoose. Now you can store the cats you create forever, and create new features that allow you to sort those cats in various ways.
+
+For this homework, you will need to add at least one feature that integrates with your database and uses an advanced query. For example, you could only show cats in a certain age range, or display cats by date created.
```

Comment:
First off, do you mean writing an explicit Mongo query, or creating a more complex query using Mongoose? I'd take "advanced" to mean using "where", "in", "and", "or", etc. predicates to combine documents across collections. https://docs.mongodb.org/manual/tutorial/query-documents/